doc/release_notes.md

```diff
@@ -4,8 +4,12 @@ Descriptions of the tagged TKey releases.
 
 ## Upcoming release: Castor
 
-Overview of changes since TK TK1-24.03 for the Castor milestone so
-far.
+Overview of changes since [TK1-24.03
+Bellatrix](https://github.com/tillitis/tillitis-key1/releases/tag/TK1-24.03)
+for the Castor milestone so far.
+
+Main branch in the repository contains Castor design and specifically
+the tag `TK1-Castor-alpha` reflects the upcoming release.
```

Comment:
I suggest tagging the release with `-1` at the end. If there are more alpha-releases later it will be clear which alpha-release was the first one.

Like so: `TK1-Castor-alpha-1`